components/Players/PlayerManager.vue

```diff
@@ -20,9 +20,15 @@
       <v-hover v-slot="{ hover }">
         <v-card class="player-card" width="100%">
           <video-player />
+          <up-next
+            v-if="showUpNext"
+            :time-left="timeLeft"
+            @hide="upNextUserHidden = true"
+          />
           <!-- Mini Player Overlay -->
           <v-fade-transition>
-            <v-overlay v-show="hover && isMinimized" absolute>
+            <!-- z-index: 5 is the default, but hardcoded in case of changes -->
+            <v-overlay v-show="hover && isMinimized" absolute z-index="5">
```

Comment:
Mostly to make sure that line 100 in `Items/UpNext.vue` which is: `z-index: 6;` is above the overlay, but it might be a bit redundant as we would probably look for visual errors when updating vuetify. I'll remove it

Reply:
Now that you're using a card component this will probably be fine.